### PR TITLE
Fix for specifying a callback as third param.

### PR DESCRIPTION
**Pull request by aroth**
Resolves https://github.com/shimohq/react-native-prompt-android/issues/4

### CODE REVIEW

File: index.android.js [hunk 1]
```diff
@@ -72,19 +72,26 @@ export default function prompt(
     callbackOrButtons?: ?((text: string) => void) | ButtonsArray,
     options?: Options
 ): void {
-    let buttons = callbackOrButtons;
+    const defaultButtons = [
+      {
```
Comment (greedbell) [inline index.android.js]: I think the cancel button should be deleted.

Comment (aroth) [inline index.android.js]: What does the Android default look like?

iOS includes `Cancel` and `OK`.

Comment (greedbell) [inline index.android.js]: but this `Cancel` button do nothing.

Comment (greedbell) [inline index.android.js]: My fault ! it is needed, and could be 

```
{
  text: 'Cancel'
}
```

Comment (aroth) [inline index.android.js]: All set, @greedbell.

**Comment by greedbell** [comment]
Thanks

**Pull request merged**